views/webrtc-view/src/components/UserGrid/Item/Item.module.css

```
@@ -158,6 +162,16 @@
   aspect-ratio: 1/1;
 }
 
+:-webkit-full-screen .item {
```

Stylelint (17.3.0)

[error] Unexpected vendor-prefixed selector ":-webkit-full-screen" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)